### PR TITLE
Completed prep-javascript-conditionals Exercise. -Matthew Hu

### PR DESCRIPTION
**Pull request by matthewkhu**
Screenshot 2024-01-16 at 5 43 39 PM

### CODE REVIEW

File: prep-javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  } else if (score < 100) {
+    return 'A';
+  } else {
+    return 'A++';
+  }
+}
+console.log('Grade', gradeCalculator(student2Score));
+
+const dayOfTheWeek = 'thursday';
+
```
Comment (shawnkost) [inline prep-javascript-conditionals/main.js]: You are missing the function `seasonMessenger`.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looks like one function was accidently missed here! Go back to the instructions and complete the missing function

**Comment by matthewkhu** [comment]
image

**Review by shawnkost — APPROVED** [review]
💯

**Pull request merged**